build/params_buildernet.go

```diff
@@ -0,0 +1,121 @@
+//go:build buildernet
+// +build buildernet
+
+package build
+
+import (
+	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/network"
+	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
+
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+)
+
+const Consensus ConsensusType = Mir
+
+// FIXME: Using devnet actors and genesis for now.
+const BootstrappersFile = ""
+const GenesisFile = ""
+
+var NetworkBundle = "devnet-wasm"
+var BundleOverrides map[actors.Version]string
+var ActorDebugging = false
+
+// var NetworkBundle = "wallaby"
```

Comment:
Should we remove these lines?

Reply:
They are a reminder for when we actually implement this issue: https://github.com/consensus-shipyard/lotus/issues/13. I rather keep them like this as a reminder for now:

Reply:
Not obvious that they are a reminder. Maybe you should add TODO FIXME?

Reply:
Feel free to resolve it without any additional comments.

Reply:
Fixed